cmd/scollector/collectors/iterable.go

```diff
@@ -2,68 +2,92 @@ package collectors
 
 import (
 	"context"
-	"fmt"
 	"time"
 
 	"bosun.org/cmd/scollector/conf"
 	"bosun.org/metadata"
 	"bosun.org/opentsdb"
+	"bosun.org/slog"
 	"github.com/bosun-monitor/statusio"
 )
 
+// iterable.io collector
+// to use add this in your scollector.toml:
+// [Iterable]
```

Comment:
Would that make it a more generic StatusPageIO collector? Seems reasonable. Config would have a `map[string]string` perhaps, mapping component name to tag name. Or can we even just automatically add cleaned tags for each component we find?